### PR TITLE
Add support for AWS_PROFILE

### PR DESCRIPTION
**Pull request by Julian**
This changeset stacks after #52 and #53.

### CODE REVIEW

**Comment by Julian** [comment]
Closes #15.

File: txaws/newsfragments/54.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+txAWS now supports loading different sets of credentials via AWS_PROFILE.
```
Comment (exarkun) [inline txaws/newsfragments/54.feature]: How about `txaws.credentials.AWSCredentials now supports ...`?

Comment (Julian) [inline txaws/newsfragments/54.feature]: Done.

**Comment by Julian** [comment]
Thanks! -- Made the wording change there.

**Pull request merged**